packages/apollo-server-fastify/src/__tests__/ApolloServer.test.ts

```diff
@@ -76,13 +76,13 @@ describe('apollo-server-fastify', () => {
     if (httpServer) await httpServer.close();
   });
 
-  describe('constructor', async () => {
+  describe('constructor', () => {
```

Comment:
Whoops, yeah, I don't know how this ever behaved.  But glad that Jest v24 warns.

Reply:
Despite the warning, the test does seem to run at present.

I wonder if there's an easy way for CI to consistently notice warnings.

Reply:
Oh probably because async functions run synchronously until their first await?

Reply:
I think that Jest struggles with an `async` `describe` because it uses a fresh worker for each `it` test and thus those tests aren't able to comply with the unknown `async`-properties of its parent `describe`?

Reply:
I think it's simpler — it just isn't interested in building the list of tests asynchronously. So it won't wait on any Promises returned from a `describe` function, but anything that happens to happen inside the function before returning a Promise still happens.  Since there are no awaits, the tests were running fine but new Jest was warning about it.